assets/docs/tulasiram58827/craft-text-detector/dr/1.md

```diff
@@ -0,0 +1,27 @@
+### Lite tulasiram58827/craft-text-detector/dr/1
```

Comment:
First line of the MD files should start with a single `#` charcter, for example a valid first line for this file would be:
`# Lite tulasiram58827/craft-text-detector/dr/1`

Same for the rest of the files in this PR.

Reply:
Done.